ntcore/src/main/native/cpp/NetworkServer.cpp

```diff
@@ -360,6 +361,17 @@ void NetworkServer::LoadPersistent() {
         "could not open persistent file '{}': {} "
         "(this can be ignored if you aren't expecting persistent values)",
         m_persistentFilename, ec.message());
+    // backup file if it exists
+    try {
+      std::filesystem::copy_file(
+          m_persistentFilename, m_persistentFilename + ".bak",
+          std::filesystem::copy_options::overwrite_existing);
```

Comment:
Use `fs::` instead of the full `std::filesystem`.  Pass `ec` as a 4th parameter to avoid the try/catch entirely.